### ISSUE TITLE
Extension coloring in Light Theme on selection is off

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#150006]
1. Code insiders, new user data dir
2.  choose Light+ theme
3. Open extensions viewlet, click on an extension

Notice color contrast is off. The text can not be readable when an extension has selection and focus

fyi @sandy081 

![Screenshot 2022-05-20 at 10 29 50](https://user-images.githubusercontent.com/1926584/169487979-dbc2c215-30e4-406e-bf21-ab7cad9e6f1c.png)

**Comment by sandy081** [comment]
It seems description color was introduced by @daviddossett. Hence assigning to him.

**Comment by daviddossett** [comment]
Yeah looks like the new color isn't getting overridden in a focused/selected state. Thought I fixed this but I'll take a look 👍
